app/api/dao/mentorship_relation.py

```diff
@@ -215,6 +215,8 @@ def cancel_relation(user_id, relation_id):
 
         # All was checked
         request.state = MentorshipRelationState.CANCELLED
+        if data and 'cancellation_reason' in data:
```

Comment:
@bhawesh96 before doing this validate the field's length. You can see an example of a function here: https://github.com/systers/mentorship-backend/blob/develop/app/api/validations/user.py#L168
See where that function is being used, if in DAO or on the API resources, follow what is being done with that function. This validating is to avoid that someone adds a cancellation reason bigger than what the model is expected. I think you're expecting 400 character limit.